src/main/java/org/octri/clinchit/domain/projections/ClinicianReviewSummary.java

```diff
@@ -0,0 +1,33 @@
+package org.octri.clinchit.domain.projections;
+
+import org.octri.clinchit.domain.ClinicianReview;
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.data.rest.core.config.Projection;
+
+/**
+ * Summarize the Clinician Review in a more natural way for REST access and the SDH view.
+ * 
+ * @author yateam
+ *
+ */
+@Projection(name = "summary", types = ClinicianReview.class)
+public interface ClinicianReviewSummary {
```

Comment:
Clinician Reviews tie together the related entities Patient, Domain, ClinicianPriority, and PatientReadiness. The default REST response provides links for you to get all the related entities by making additional calls (e.g. "/api/clinician_review/1/domain"). Using the Projection strategy, we can just get the entities or fields we want in a single call.

Reply:
Good find on these Projections. I can see them being pretty useful.